cn.navclub.ldbfx.appdata.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>cn.navclub.ldbfx</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>Apache-2.0</project_license>
+  <name>ldbfx</name>
+  <summary>Visual database management</summary>
+  <description>A visual database management tool for Linux.</description>
+  <url type="homepage">https://github.com/databasefx/ldbfx</url>
+  <url type="bugtracker">https://github.com/databasefx/ldbfx/issues</url>
+  <screenshots>
+    <screenshot>
+      <image>https://github.com/databasefx/ldbfx/blob/master/SNAPSHOT/home.png</image>
+    </screenshot>
+    <screenshot>
+      <image>https://github.com/databasefx/ldbfx/blob/master/SNAPSHOT/main.png</image>
+    </screenshot>
+  </screenshots>
+  <update_contact>752544765@qq.com</update_contact>
```

Comment:
as I mentionned before the appdata is incomplete.

It is missing at least `content_rating`